code/modules/reagents/chemistry/reagents/medicine_reagents.dm

```diff
@@ -1057,6 +1058,21 @@
 	REMOVE_TRAIT(M, TRAIT_STABLEHEART, type)
 	REMOVE_TRAIT(M, TRAIT_STABLELIVER, type)
 	..()
+/datum/reagent/medicine/penthrite
+	// Same as corazone. but slightly better
+	name = "penthrite"
+	description = "A medication used to treat certain heart conditions, also it is explosive"
```

Comment:
Awkward wording, perhaps "A medication used to treat certain heart conditions, it is also explosive." or "An explosive compound used to treat certain heart conditions." etc